src/sql-parser/src/ast/defs/statement.rs

```diff
@@ -1924,6 +1927,27 @@ impl AstDisplay for AlterObjectRenameStatement {
 }
 impl_display!(AlterObjectRenameStatement);
 
+/// `ALTER <OBJECT> SWAP ...`
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+pub struct AlterObjectSwapStatement {
+    pub object_type: ObjectType,
+    pub name_a: UnresolvedObjectName,
+    pub name_b: UnresolvedObjectName,
```

Comment:
a naming idea: before vs after. feel free to take it or leave it

Reply:
Thanks for the suggestion! Since it's a swap there isn't really a before and after, which is why I went with 'a' and 'b'